src/MovieCard/MovieCard.css

```diff
@@ -33,6 +33,11 @@
   -12px -12px 50px 1px rgba(194, 3, 252, .4);
 }
 
+.card-image:hover {
+  margin: 0% 5% 0% 5%;
+  object-fit: cover;
+}
+
```

Comment:
I took this out on the last PR, I realized there was a hover on the card container and the image, and  I think it is too much hover action on the card and feels a bit glitchy. Looks like it's being added back in here but taken out below in the media queries

Reply:
Ok it was on other screen sizes so I thought so was a little unsure I will pull it out and let you know when its done.

Reply:
Fixed

Reply:
Yeah I totally didn't remove it from the media queries and they should be so that makes sense.